charts/nextcloud/templates/nginx-config.yaml

```diff
@@ -44,7 +44,7 @@ data:
         }
 
         server {
-            listen 80;
+            listen {{ .Values.nextcloud.containerPort | default 80 }};
```

Comment:
Could you move the default value into the values file? That would make it easier to reuse for https://github.com/nextcloud/helm/pull/316

Reply:
Yes